fief/settings.py

```diff
@@ -124,6 +124,7 @@ class Settings(BaseSettings):
 
     class Config:
         env_file = ".env"
+        secrets_dir = '/run/secrets'
```

Comment:
I was thinking is this could be read from an environment variable, with a default to `/run/secrets`. I know that there are some cloud providers/deployment tech that also use this secret pattern, but with a different folder path. This would allow them to set it dynamically to fit their need. Something like:

```
secrets_dir = os.environ.get("SECRETS_DIR", "/run/secrets")
```

PS: the linter complains because of the single quotes.

Reply:
Yes, that makes sense.

We can also add an "InitialSettings" pydantic settings. This way we still can have all the power of pydantic.

like:
```
class InitialSettings(BaseSettings):
      secrets_dir:str="/run/secrets"

initial_settings = InitialSettings()

class Settings(BaseSettings):
      Config:
             secrets_dir = initial_settings.secrets_dir
```
PS: I was wondering why the linter failed :).